Sources/App/Settings/SettingsRootDataSource.swift

```diff
@@ -95,6 +97,16 @@ enum SettingsRootDataSource {
         }
     }
 
+    private static func security() -> SettingsButtonRow {
+        SettingsButtonRow {
+            $0.title = "Security"
```

Comment:
you can wrap in `NSLocalizedString` as a reminder to add it to the L10n/swiftgen stuff, makes it a little easier to avoid missing things

Reply:
How does the translation flow works? Like... I can input English and Portuguese, does the community fills the rest?

Reply:
I have wrapped all strings I added for now